libp2p/protocols/pubsub/pubsubpeer.nim

```diff
@@ -67,6 +67,8 @@ type
   PubSubObserver* = ref object
     onRecv*: proc(peer: PubSubPeer, msgs: var RPCMsg) {.gcsafe, raises: [].}
     onSend*: proc(peer: PubSubPeer, msgs: var RPCMsg) {.gcsafe, raises: [].}
+    onRecvAndValidated*:
```

Comment:
How about `onValidated`?

Reply:
Perfect! Renamed it. Had the `onRecv` part to be more explicit that it was only when messages are received (not sure if we run validations before sending messages). But if `onValidated` is clear enough then even better :)